third_party/xla_client/pjrt_computation_client.cc

```diff
@@ -114,31 +117,38 @@ std::vector<ComputationClient::DataPtr> PjRtComputationClient::TransferToServer(
         std::make_shared<PjRtData>(tensor.device, tensor.shape, buffer);
     datas.push_back(data);
   }
+  OutboundDataMetric()->AddSample(total_size);
+  CreateDataHandlesCounter()->AddValue(datas.size());
```

Comment:
@JackCaoG It looks like PjRt doesn't have the same notion of "handles" as XRT. I'm not familiar with handles in XRT, so I wanted to get some clarification - do these "handle" metrics still make sense to use in PjRt, or should we make a new counter?

Reply:
yea handle is same as buffer in the `TransferToServer` context.